docs/UserGuide.md

```diff
@@ -62,6 +64,30 @@ It allows for faster and more effective student management.
 
 ## Features
 
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes about the command format:**<br>
+
+* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
+  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+
+* Items in square brackets are optional.<br>
+  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
```

Comment:
Hmmmm actually using TAG in this case might not be intuitive as we do not have a tag parameter in our entire application. If we are to include this here, maybe a different parameter and including the edit_student in the UG would then be more useful.

Reply:
Whoops was copied wholesale 😂 will make the change, thanks!

Reply:
Actually on second thought, let's just scrap this optional explanation until we have a command with optionals.